scripts/run-local-exec

```diff
@@ -16,9 +16,9 @@ do
 done
 
 if [[ "$PUBLISHED" == "true" ]]; then
-  EXE="backend/Build/out/ExecHost/Release/net7.0/linux-x64/publish/ExecHost"
+  EXE="backend/Build/out/LocalExec/Release/net7.0/linux-x64/publish/LocalExec"
 else
-  EXE="backend/Build/out/ExecHost/Debug/net7.0/linux-x64/ExecHost"
+  EXE="backend/Build/out/LocalExec/Debug/net7.0/LocalExec"
```

Comment:
a bit old, but I just noticed that this omits the 'linux-x64' part of the directory, which seems inconsistent with other references. Just raising the point in case it was a mistake

Reply:
That got removed from a bunch of places as we no longer only build on linux-x64 and so .NET doesn't include the directory when emitting the output executables.

So I think this was intentional